app/src/main/java/com/nervousfish/nervousfish/data_objects/Database.java

```diff
@@ -0,0 +1,107 @@
+package com.nervousfish.nervousfish.data_objects;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+
+import java.io.InvalidObjectException;
+import java.io.ObjectInputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A database object to write and encrypt to file
+ */
+public final class Database implements Serializable {
+    private static final long serialVersionUID = -4715364587956219157L;
+    private final List<Contact> contacts;
+    private Profile profile;
+
+
+    /**
+     * The constructor for the {@link Database} POJO
+     * @param contacts The contacts the user has.
+     * @param profile The profile of the user.
+     */
+    public Database(final List<Contact> contacts, final Profile profile) {
+        this.contacts = contacts;
+        this.profile = profile;
+    }
+
+
+    /**
+     * Getter for the contacts.
+     * @return The contacts list.
+     */
+    public List<Contact> getContacts() {
+        return this.contacts;
+    }
+
+    /**
+     * Sets the list of contacts to the given list of contacts.
+     * @param contacts  The new list of contacts.
+     */
+    public void setContacts(final List<Contact> contacts) {
+        this.contacts.clear();
+        this.contacts.addAll(contacts);
+    }
+
+    /**
+     * Getter for the user profile
+     * @return  The User profile.
+     */
+    public Profile getProfile() {
+        return this.profile;
+    }
+
+    /**
+     * Sets the user's profile to a new profile
+     * @param profile The new profile.
+     */
+    public void setProfile(final Profile profile) { this.profile = profile; }
+
+    /**
+     * Serialize the created proxy instead of this instance.
+     */
+    private Object writeReplace() {
+        return new SerializationProxy(this);
```

Comment:
Should be Database.SerializationProxy, because that's the code style that's mostly used in this project (apparently). Press Ctrl-Alt-S -> Editor -> Inspections -> Manage -> Import. https://www.dropbox.com/s/wryiggq2x9yr0tz/Project_Default.xml?dl=0
Hover over SerializationProxy and press Cleanup Code. That will automatically fix all "style errors" in the whole document of that type